client/my-sites/plans/jetpack-plans/has-standalone-plugin.ts

```diff
@@ -0,0 +1,28 @@
+import {
+	JETPACK_BACKUP_PRODUCTS,
+	JETPACK_BOOST_PRODUCTS,
+	JETPACK_CRM_PRODUCTS,
+	JETPACK_SEARCH_PRODUCTS,
+	JETPACK_SOCIAL_PRODUCTS,
+} from '@automattic/calypso-products';
+
+const ALL_PRODUCTS_WITH_STANDALONE_PLUGIN = [
+	...JETPACK_BACKUP_PRODUCTS,
+	...JETPACK_BOOST_PRODUCTS,
+	...JETPACK_SOCIAL_PRODUCTS,
+	...JETPACK_CRM_PRODUCTS,
+	...JETPACK_SEARCH_PRODUCTS,
+] as ReadonlyArray< string >;
+
+/**
+ * Determines if a product slug has a standalone plugin.
+ *
+ * @example
+ * jetpack_backup_daily	      > true
+ * jetpack_security_t1_yearly > false
+ * @param {string} productSlug Product slug
+ * @returns {boolean} Parameters
+ */
+export default ( productSlug: string ): boolean => {
```

Comment:
Can we please avoid un-named functions?

Also, I think its name should be `isJetpackStandaloneProduct` and likewise the file name and function doc block.

Reply:
Thanks for spotting this @manzoorwanijk . yes I agree the suggested name has better context. I updated the file name and as well documentations. I also renamed some few variable names to match the new changes.